tsconfig.json

```diff
@@ -23,7 +23,8 @@
     },
     "typeRoots": [
       "./src/types",
-      "./node_modules/@types/"
+      "./node_modules/@types/",
+      "./node_modules/"
```

Comment:
suggestion: I would recommend to remove `node_modules/@types` in this case.

Reply:
That doesn't work as then he wont find the rest of the type definitions anymore.